Tryouts/Prototypes/Shell/MainWebWindowOptionsParser.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Binding;
+using System.CommandLine.Parsing;
+using System.Linq;
+using System.Security.Policy;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media.Media3D;
+
+namespace Shell
+{
+    internal class MainWebWindowOptionsParser
+    {
+        private Option<string> titleOption = new Option<string>("--title", description: "Set title for window");
+        private Option<string> urlOption = new Option<string>("--url", description: "Set url for webview. default: about:blank");
+        private Option<int> widthOption = new Option<int>("--width", description: "Set width for window");
+        private Option<int> heightOption = new Option<int>("--height", description: "Set height for window");
+        private RootCommand rootCommand = new RootCommand();
+
+        public MainWebWindowOptionsParser() {
+            rootCommand.Add(titleOption);
+            rootCommand.Add(urlOption);
+            rootCommand.Add(widthOption);
+            rootCommand.Add(heightOption);
+        }
+
+        public MainWebWindowOptions Parse(string[] args)
+        {
+            Parser parser = new Parser(rootCommand);
+            ParseResult parseResult = parser.Parse(args);
+
+            MainWebWindowOptions options = new MainWebWindowOptions
+            {
+                Title = parseResult.GetValueForOption(titleOption) ?? "Compose Web Container",
```

Comment:
Some tests would be useful, I'm not sure what value is returned when the option is not set (it might even be an exception that we probably don't want)